baybe/searchspace/discrete.py

```diff
@@ -134,19 +132,20 @@ def __str__(self) -> str:
         dont_recommend_count = len(self.metadata[self.metadata[_METADATA_COLUMNS[2]]])
         metadata_count = len(self.metadata)
 
-        # Put all attributes of the discrete class in one string.
-        discrete_str = f"""{start_bold}Discrete Search Space{end_bold}
-            \n{start_bold}Discrete Parameters{end_bold}\n{pretty_print_df(param_df)}
-            \n{start_bold}Experimental Representation{end_bold}
-            \r{pretty_print_df(self.exp_rep)}\n\n{start_bold}Metadata:{end_bold}
-            \r{_METADATA_COLUMNS[0]}: {was_recommended_count}/{metadata_count}
-            \r{_METADATA_COLUMNS[1]}: {was_measured_count}/{metadata_count}
-            \r{_METADATA_COLUMNS[2]}: {dont_recommend_count}/{metadata_count}
-            \n{start_bold}Constraints{end_bold}\n{pretty_print_df(constraints_df)}
-            \n{start_bold}Computational Representation{end_bold}
-            \r{pretty_print_df(self.comp_rep)}"""
-
-        return discrete_str.replace("\n", "\n ").replace("\r", "\r ")
+        metadata = f"""Metadata:
+            {_METADATA_COLUMNS[0]}: {was_recommended_count}/{metadata_count}
```

Comment:
the triple `"""` does not produce the indent you wish for

Reply:
Will investigate and adjust.